examples/fabric-admin/commands/common/IcdManager.cpp

```diff
@@ -32,6 +34,20 @@ void IcdManager::OnCheckInComplete(const chip::app::ICDClientInfo & clientInfo)
     {
         mDelegate->OnCheckInCompleted(clientInfo);
     }
+
+    uint32_t activeDurationMs = 30000;
+    auto stayActiveSender = chip::Platform::New<StayActiveSender>(activeDurationMs, clientInfo.peer_node, chip::app::InteractionModelEngine::GetInstance());
```

Comment:
On of the issues I previously mentioned is that `clientInfo.peer_node` is the node of fabric-admin, and not the LIT device checking-in